src/converters/RpcConverters.ts

```diff
@@ -28,6 +29,16 @@ export function fromTypedData(typedData?: rpc.ITypedData, convertStringToJson: b
         return str;
     } else if (typedData.bytes) {
         return Buffer.from(<Buffer>typedData.bytes);
+    } else if (typedData.collectionBytes && typedData.collectionBytes.bytes) {
+        let byteCollection = <Uint8Array[]>typedData.collectionBytes.bytes;
+        return byteCollection.map(element => Buffer.from(<Buffer>element));
+    } else if (typedData.collectionString && typedData.collectionString.string) {
+        return <string[]>typedData.collectionString.string;
+    } else if (typedData.collectionDouble && typedData.collectionDouble.double) {
+        return <number[]>typedData.collectionDouble.double;
+    } else if (typedData.collectionSint64 && typedData.collectionSint64.sint64) {
+        let longCollection = <(Long|number)[]>typedData.collectionSint64.sint64;
+        return longCollection.map(element => isLong(element) ? element.toString() : <number>element);
```

Comment:
What will happen in case the type is not defined in these if, specially there is no else default case? This case should not ever happen.

Reply:
@amamounelsayed - agreed... good catch :) I'm going to track this as an issue to fix separately, since it's an improvement on our current behavior and we do want to get this in for the release today: https://github.com/Azure/azure-functions-nodejs-worker/issues/267